Dangerfile

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+
+require 'active_support/core_ext/string/output_safety'
+
+def verifyPRTitle()
+    title = github.pr_title
+    if title.size < 17
+        warn("Title too short. It should fit '[KLTB002-JIRA_TICKET_NO] TITLE' template.")
+    else 
+        if title.start_with?("[KLTB002-")
```

Comment:
You should make a regex instead of checking the beginning, and check only for `\[PROJECT_CODE\-[0-9]+\]` ?
Or you can get the labels between brackets and check if you have any label with `PROJECT_CODE-[0-9]`.

Reply:
I was not changing the rule used in iOS repo but took it as-is. I agree, it's far from perfect.